src/components/LanguageSwitcher.js

```diff
@@ -16,14 +17,18 @@ function LanguageSwitcher() {
     i18n.changeLanguage(target.value);
   };
 
+  useEffect(() => {
```

Comment:
As a note, we will want to check for edge cases in the future. Especially, if this works in nested routes. No push needed.

Reply:
Yeah this is definitely preliminary just so I could see the language content change. We can look into best practices for multilingual routing in gatsby